3.10/aql/operations-for.md

```diff
@@ -196,6 +196,34 @@ there are a few corner cases where it can make sense:
   projections, if the cost of projections is higher than doing copies of the
   full documents. This can be the case for very small documents.
 
+### `useCache`
+
+<small>Introduced in: v3.10.0</small>
+
+When using an index is used to produce the documents for an AQL FOR loop,
+the `useCache` hint can be used to disable the usage of the in-memory cache
+for the index.
+This is useful for queries that access indexes with enabled in-memory caches,
```

Comment:
Should reference `cacheEnabled` here to make it clear that this is user-controlled. Currently, it sounds like ArangoDB would use in-memory caches for certain indexes and that the user would need to figure out which ones.